advanced/docker-compose.valkey-cluster.volumes.yml

```diff
@@ -32,42 +32,42 @@
 version: '3.0'
 
 services:
-  # Redis cluster
-  redis-node-0:
+  # Valkey cluster
+  valkey-node-0:
     volumes:
-      - redis-cluster-data-0:/bitnami/redis/data
-  redis-node-1:
+      - valkey-cluster-data-0:/bitnami/valkey/data
+  valkey-node-1:
     volumes:
-      - redis-cluster-data-1:/bitnami/redis/data
-  redis-node-2:
+      - valkey-cluster-data-1:/bitnami/valkey/data
+  valkey-node-2:
     volumes:
-      - redis-cluster-data-2:/bitnami/redis/data
-  redis-node-3:
+      - valkey-cluster-data-2:/bitnami/valkey/data
+  valkey-node-3:
     volumes:
-      - redis-cluster-data-3:/bitnami/redis/data
-  redis-node-4:
+      - valkey-cluster-data-3:/bitnami/valkey/data
+  valkey-node-4:
     volumes:
-      - redis-cluster-data-4:/bitnami/redis/data
-  redis-node-5:
+      - valkey-cluster-data-4:/bitnami/valkey/data
+  valkey-node-5:
     volumes:
-      - redis-cluster-data-5:/bitnami/redis/data
+      - valkey-cluster-data-5:/bitnami/valkey/data
 
 volumes:
-  redis-cluster-data-0:
+  valkey-cluster-data-0:
     external:
-      name: ${REDIS_CLUSTER_DATA_VOLUME_0}
-  redis-cluster-data-1:
+      name: ${VALKEY_CLUSTER_DATA_VOLUME_0}
```

Comment:
The same issue as with CE. See my original comment from CE PR:

https://github.com/smatvienko-tb/thingsboard/pull/8#discussion_r2101925536